data/json/effects.json

```diff
@@ -386,6 +386,21 @@
     },
     {
         "type": "effect_type",
+        "id": "nausea",
+        "name" : ["Nausea"],
+        "desc": ["You feel nauseous.  Even a thought of food makes you feel sick."],
+        "remove_message": "You no longer feel nauseous.",
+        "rating": "bad",
+        "resist_traits": ["STRONGSTOMACH"],
+        "int_dur_factor": 300,
+        "max_intensity": 3,
+        "max_duration": 900,
+        "base_mods": {
+            "vomit_chance": [60]
+        }
+    },
```

Comment:
In the description you say the intensity affects vomit chance, I don't see it here.